CommunityFridgeMapApi/functions/image/v1/app.py

```diff
@@ -0,0 +1,45 @@
+import json
+import base64
+
+try:
+    from storage import Storage
+except:
+    from dependencies.python.storage import Storage
+
+class ImageHandler:
+    @staticmethod
+    def get_binary_body_from_event(event: dict) -> bytes:
+        """Extract binary data from request body"""
+        assert event["isBase64Encoded"]
+        return base64.b64decode(event["body"])
+
+    @staticmethod
+    def encode_binary_file_for_response(blob: bytes) -> bytes:
+        """
+        Binary response body of lambda functions should be encoded in base64.
+        https://docs.aws.amazon.com/apigateway/latest/developerguide/lambda-proxy-binary-media.html
+        """
+        return base64.b64encode(blob)
+
+    @staticmethod
+    def lambda_handler(event: dict, storage: Storage) -> dict:
+        bucket = "fridge-report"
```

Comment:
This API handles images from fridge and fridge-report. Should we put all of the images in the same folder? or use two different folders?

If we use two different folders should the client pass in a field to differentiate between fridge and fridge-report?

Reply:
We can use a multipart form to pass-in a bucket name & binary file.
The spec only had single file upload, so that's why I implemented the lambda handler this way.

Reply:
can we name the bucket `CommunityFridgeMapImages`?

fridge-report makes it sounds like only fridge-report images will be uploaded here

Reply:
Yep, fixed here e0086626d9f64e2dec63eddd9731bbd60f149af0